packages/components/pages/f-checkout/package.json

```diff
@@ -66,16 +66,17 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "7.12.13",
-    "@justeat/f-alert": "4.2.0",
-    "@justeat/f-button": "3.3.0",
-    "@justeat/f-card": "3.6.0",
-    "@justeat/f-card-with-content": "1.1.0",
-    "@justeat/f-error-message": "1.1.0",
+    "@justeat/f-alert": "6.0.0",
+    "@justeat/f-button": "4.2.0",
+    "@justeat/f-card": "4.1.0",
+    "@justeat/f-card-with-content": "3.0.0",
+    "@justeat/f-error-message": "2.1.0",
     "@justeat/f-form-field": "4.9.0",
```

Comment:
not updating this because the asterisks changes to `f-form-field` probably warrant a good combing through which should be its own ticket